x-pack/plugin/ml/qa/native-multi-node-tests/src/test/java/org/elasticsearch/xpack/ml/integration/MlNativeAutodetectIntegTestCase.java

```diff
@@ -347,6 +348,7 @@ protected void waitForecastToFinish(String jobId, String forecastId) throws Exce
         }, 30, TimeUnit.SECONDS);
     }
 
+    //TODO: 6.7 Adjust so that multiple indices are supported
```

Comment:
All 3 places where you have `TODO: 6.7 Adjust so that multiple indices are supported` should be changed before this PR is merged, as this functionality will stop working when migration creates a second results index.

Reply:
@droberts195 absolutely correct :D, I put those there to remind me after break that it was something I needed to do! Thanks for calling it out.

https://github.com/elastic/elasticsearch/pull/37157